libp2pdht/private/eth/p2p/discoveryv5/protocol.nim

```diff
@@ -1023,76 +1032,6 @@ func init*(
     bitsPerHop: bitsPerHop
   )
 
-proc newProtocol*(
```

Comment:
not clear to me why this was removed. Maybe right to do so, but I don't know why